Resources/Prototypes/Recipes/Construction/structures.yml

```diff
@@ -322,38 +322,98 @@
   placementMode: SnapgridCenter
   canBuildInImpassable: false
   conditions:
-  - !type:TileNotBlocked {}
+    - !type:TileNotBlocked
 
+
+#lighting
 - type: construction
-  name: railing
-  id: Railing
-  graph: railing
+  name: wall light
+  id: LightTubeFixture
+  graph: lightFixture
   startNode: start
-  targetNode: railing
+  targetNode: tubeLight
   category: Structures
-  description: Basic railing meant to protect idiots like you from falling.
+  description: A wall light fixture. Use light tubes.
   icon:
-    sprite: Structures/Walls/railing.rsi
-    state: side
+    sprite: Structures/Wallmounts/Lighting/light_tube.rsi
+    state: off
   objectType: Structure
   placementMode: SnapgridCenter
+  canRotate: true
   canBuildInImpassable: false
   conditions:
-  - !type:TileNotBlocked {}
+  # Need an *additional* condition here that forces there to be a wall in the opposite direction to the one used for placement.
+  # Also see below. Didn't add it b/c construction ECS work going on. Cheers, - 20kdc
+    - !type:TileNotBlocked
 
 - type: construction
-  name: railing corner
-  id: RailingCorner
-  graph: railing
+  name: small wall light
+  id: LightSmallFixture
+  graph: lightFixture
   startNode: start
-  targetNode: railingCorner
+  targetNode: bulbLight
   category: Structures
-  description: Basic railing meant to protect idiots like you from falling.
+  description: A wall light fixture. Use light bulbs.
   icon:
-    sprite: Structures/Walls/railing.rsi
-    state: corner
+    sprite: Structures/Wallmounts/Lighting/light_small.rsi
+    state: off
   objectType: Structure
   placementMode: SnapgridCenter
+  canRotate: true
   canBuildInImpassable: false
   conditions:
-  - !type:TileNotBlocked {}
```

Comment:
afaict railing & railingcorner are gone?

Reply:
Not sure what happened here if railings got merged before and I messed up the conflict or I just messed up, will correct tomorrow. Thanks,

Reply:
Can't merge master on my PR without it breaking my git but I re-added railings under barricade. They were merged before this once hence conflict that I messed.